tests/test_convert.py

```diff
@@ -14,7 +14,7 @@ def out_file():
         yield out
 
 
-@mark.parametrize('converter', ['at', 'be_vlg', 'br_ba_lem', 'de_sh', 'ec_lv', 'ec_si', 'fi', 'fr', 'nl', 'nl_crop', 'pt'])
```

Comment:
Is the removal of br_ba_lem intentional?

Reply:
Thanks for spotting this, a rebase/merge mistake